src/SIL.Machine.AspNetCore/Services/ClearMLService.cs

```diff
@@ -12,10 +13,19 @@ public class ClearMLService : IClearMLService
             Converters = { new CustomEnumConverterFactory(JsonNamingPolicy) }
         };
 
-    public ClearMLService(HttpClient httpClient, IOptionsMonitor<ClearMLNmtEngineOptions> options)
+    private string authorizationToken = "";
```

Comment:
These fields should be prefixed with underscores.

Reply:
Done.